pallet-logion-vote/src/tests.rs

```diff
@@ -21,6 +22,7 @@ fn it_creates_vote() {
                 ]
             }));
         assert_eq!(LogionVote::votes(2), None);
+        assert_eq!(LogionVote::is_vote_completed(vote_id), (false, false))
```

Comment:
Is there a way to check that an expected event has actually been deposited? As we are heavily relying here on events, it would be nice to explicitly check their deposit. If not trivial nor easy, let's keep this for later in the form of a technical debt.

Reply:
It is possible to test emitted events, this is done in frame pallets tests. But (so far) I have not been able to do the same for our own pallets.